Source/Core/VideoCommon/VideoConfig.h

```diff
@@ -32,14 +32,14 @@ enum class AspectMode
   Stretch,
 };
 
-enum StereoMode
+enum class StereoMode
 {
-  STEREO_OFF = 0,
-  STEREO_SBS,
-  STEREO_TAB,
-  STEREO_ANAGLYPH,
-  STEREO_QUADBUFFER,
-  STEREO_3DVISION
+  Off,
+  SBS,
+  TAB,
+  Anaglyph,
+  QuadBuffer,
+  ThreeDVision
```

Comment:
Could we call this `Nvidia3DVision` so that we don't have to spell 3 using letters?

Reply:
Sure thing

Reply:
Done